src/components/Balatro/Balatro.tsx

```
@@ -0,0 +1,237 @@
+    pixelFilter?: number;
+    spinEase?: number;
+    isRotate?: boolean;
+    mouseInteraction?: boolean;
+}
+
+function hexToVec4(hex: string): [number, number, number, number] {
+    const hexStr = hex.replace("#", "");
+    let r = 0,
+        g = 0,
+        b = 0,
+        a = 1;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)